data_structures/is_multiple.py

```diff
@@ -0,0 +1,17 @@
+# is multiple of a number.
+# numbers n and m, where n = mi.
+# return true if n is a multiple of m else return false.
+
+
+def is_multiple(n, m):
```

Comment:
Shouldn't this return a `True` or `False`

Reply:
return value corrected as True or False.